components/spreadsheet_com/actions/update-rows/update-rows.mjs

```diff
@@ -0,0 +1,53 @@
+import { parseArray } from "../../common/utils.mjs";
+import spreadsheetCom from "../../spreadsheet_com.app.mjs";
+
+export default {
+  key: "spreadsheet_com-update-rows",
+  name: "Update Rows",
+  version: "0.0.1",
+  description: "Update existing row(s). [See the documentation](https://developer.spreadsheet.com/#tag/Rows/operation/updateRows)",
+  type: "action",
+  props: {
+    spreadsheetCom,
+    workbookId: {
+      propDefinition: [
+        spreadsheetCom,
+        "workbookId",
+      ],
+    },
+    worksheetId: {
+      propDefinition: [
+        spreadsheetCom,
+        "worksheetId",
+        ({ workbookId }) => ({
+          workbookId,
+        }),
+      ],
+    },
+    rowsData: {
+      type: "string[]",
+      label: "Rows Data",
+      description: "Data row(s) to update. Response will contain list of row ids for which the operation was successful. Maximum of 100 entries can be provided. [Where to find row Ids](https://developer.spreadsheet.com/#section/Using-the-API/Extract-Resource-IDs)",
+    },
```

Comment:
I'll leave it up to you and/or QA, but do you think it might be a better user experience if we had the rowIds available as async options? You could then have additionalProps to display the appropriate number of "string" props for the row data.